docs/Changelog.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
```

Comment:
Still working on formatting. Not showing up correctly on localdev, but the content should be good.